conf/global.json

```diff
@@ -35,6 +35,9 @@
         "read_capacity": 20,
         "write_capacity": 5
       }
+    },
+    "classifier_sqs": {
```

Comment:
we might want to actually put this in the `rules_engine` config in `lambda.json` instead

Reply:
So I was thinking about that—I wasn't sure about the conventions on what goes in `lambda.json` and what goes in `globals.json`

I chose here because this resource isn't _exactly_ relegated to the rules engine; it also is connected to the classifier. An alternative was to put a configuration under the `classifier` AND the `rules_engine` but that would duplicate data, giving an opportunity to screw things up

Reply:
yeah I think this is fine since it also effects a 'global' resource. feel free to keep it